loky/backend/semaphore_tracker.py

```diff
@@ -73,11 +72,13 @@ def ensure_running(self):
                 # process will out live us, so no need to wait on pid
                 exe = spawn.get_executable()
                 args = [exe] + util._args_from_interpreter_flags()
-                if len(args[-1]) > 100:  # pragma: no cover
-                    warnings.warn(
-                        "args_from_flag returned -R+ -> why??", RuntimeWarning)
```

Comment:
Indeed. I relaunched travis. I don't know what we can possibly do to debug that issue. Maybe we could add a line such as the following in the travis configuration to ensure that it's possible to launch a subprocess on the host before running pytest in tox?

```
...
commands =
     python continuous_integration/install_coverage_subprocess_pth.py
     python -c "import subprocess as sp, sys; sp.Popen([sys.executable, '--version'])"
     py.test {posargs:-lv --maxfail=2 --timeout=10}
     coverage combine --append

```